apps/src/code-studio/LinkCleverAccountModal.jsx

```diff
@@ -93,7 +93,7 @@ export default class LinkCleverAccountModal extends React.Component {
         fixedWidth={600}
         fixedHeight={310}
         isOpen={this.props.isOpen}
-        handleClose={this.cancel}
+        uncloseable
```

Comment:
Bug #2

Reply:
:+1: I can confirm that `uncloseable` doesn't just remove the 'x' button, it also removes the ability to close by clicking outside the dialog or pressing the <kbd>Esc</kbd> key.